examples/Demo/Shared/Microsoft.FluentUI.AspNetCore.Components.xml

```diff
@@ -4405,6 +4406,8 @@
             </summary>
             <remarks>
             This method requires dynamic access to code. This code may be removed by the trimmer.
+            If the assembly is not yet loaded, it will be loaded by the method `Assembly.Load`.
```

Comment:
It appears that building automatically updates this file - the documentation was missing in dev.